hack/local-up-cluster.sh

```diff
@@ -800,7 +800,7 @@ function start_kubelet {
       all_kubelet_flags+=(--containerized)
 
       docker run --rm --name kubelet \
-        --volume=/:/rootfs:ro,rslave \
+        --volume=/:/rootfs:rw,rslave \
```

Comment:
does this mean existing deployments will break if they upgrade to this fix and don't change this setting?

Previously, safemkdir still worked for non-hostpath types because it uses /var/lib/kubelet/.... instead of /rootfs/var/lib/kubelet

Reply:
That's right, it will break.

I could pass `podDir` into `SafeMakeDir()` as a new parameter. `NsenterMounter.SafeMakeDir` could then check if the directory to create is subdir of `podDir` and add `/rootfs` only when it's not.

Any better ideas?

Reply:
I added a new argument to `NewNsenterMounter` so the mounter knows what's location of `/var/lib/kubelet` and can do a shortcut in `SafeMakeDir`. That was the easiest way I could find out. PTAL.

Reply:
And I removed changes in hack/local-up-cluster.sh